@here/harp-geojson-datasource/lib/GeoJsonDataProvider.ts

```diff
@@ -10,6 +10,19 @@ import { TileKey } from "@here/harp-geoutils";
 import { ConcurrentTilerFacade } from "@here/harp-mapview";
 import { DataProvider } from "@here/harp-mapview-decoder";
 
+export interface GeoJsonDataProviderOptions {
+    /**
+     * WorkerTilerUrl Worker tiler URL. Defaults to `./decoder.bundle.ts` in the
+     * [[ConcurrentTilerFacade]].
+     */
+    workerTilerUrl?: string;
+
+    /**
+     * tiler Specified tiler to be used instead of using default Tiler with Web Workers
```

Comment:
Reword. Example:

Custom tiler instance.

If not provided, [[GeoJsonDataProvider]] will obtain [[WorkerBasedTiler]] from [[ConcurrentTilerFacade]].